src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -99,6 +99,35 @@ GARY_IMPL(VMHELPDEF, hlpDynamicFuncTable, DYNAMIC_CORINFO_HELP_COUNT);
 
 #else // DACCESS_COMPILE
 
+long g_cbILJitted = 0;
+int g_cMethodsJitted = 0;
+
+long GetJittedBytes()
```

Comment:
These methods have to be defined as FCALLs or QCALLs. 

https://github.com/dotnet/runtime/blob/master/docs/design/coreclr/botr/corelib.md